packages/backend-plugin-api/report.api.md

```diff
@@ -24,6 +25,76 @@ import { Readable } from 'stream';
 import { Request as Request_2 } from 'express';
 import { Response as Response_2 } from 'express';
 
+// @public
+export type AuditorEvent = [
```

Comment:
Reminder to remove this and related types that aren't references by `AuditorService`